tests/Unit/Repositories/DatabaseRepositoryTest.php

```diff
@@ -9,6 +9,14 @@
 
 class DatabaseRepositoryTest extends TestCase
 {
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        // services.mailgun.secret is no longer a default config in Laravel 11
+        config()->set('services.mailgun.secret', null);
+    }
+
```

Comment:
I would change the Mailgun secret in that case with another default config key from Laravel

Reply:
why? What does using a default config actually contribute to the test? If anything you need knowledge that this is a default config variable in Laravel in order to understand the test. I would personally even think about using all random keys in the test.

Reply:
The test name suggests 'it can overwrite settings from default files' - so I guess to match the name of the test, it would be more logical to use a different (default) key. 
Testing random keys does seem like a logic test aswell but this is being tested in the `it_can_overwrite_settings_from_default_files_with_custom_config_keys` and `it_can_overwrite_settings_from_custom_config_files` if I'm reading correctly

Reply:
I somehow didn't even notice the `it_can_overwrite_settings_from_custom_config_files` basically testing the same thing as `it_can_overwrite_settings_from_default_files_with_custom_config_keys`. I actually propose we make this one test called `it_can_overwrite_settings_from_config` as that is what we want to test. We're not testing the Laravel Config Repository. This comes with Laravel and I'm assuming is tested in Laravel, so in my eyes it doesn't and shouldn't matter where it's values come from. All we want to test is that, when the Config has a value our repository can overwrite it with a value from the database.